ros2pkg/ros2pkg/resource/cpp/header.hpp.em

```diff
@@ -0,0 +1,19 @@
+#ifndef @(package_name.upper())__@(library_name.upper())__HPP_
```

Comment:
I don't think this will pass linting. There should be only a single underscore before `HPP`.